src/nvim/ex_docmd.c

```diff
@@ -6788,27 +6801,51 @@ void free_cd_dir(void)
 
 #endif
 
-/*
- * Deal with the side effects of changing the current directory.
- * When "local" is TRUE then this was after an ":lcd" command.
- */
-void post_chdir(int local)
+/// Deal with the side effects of changing the current directory.
+///
+/// @param scope  Scope of the function call (editor, tab or window).
+void post_chdir(int scope)
```

Comment:
I ran
~~~~
rm -r build
make clean
make distclean
make
~~~~
and still got the error when using an enum as the argument type.
http://pastebin.com/ZcDMbNgx